### ISSUE TITLE
Cannot log into KDE Identities from within a Container

### ISSUE THREAD

**Issue by Dngrsone** [mozilla/multi-account-containers#1341]
- Is "Firefox will: Never remember history" in the Firefox Preferences/Options under "Privacy & Security > History" selected? **No**
- Are you using Firefox in a Private Window? **No**
- Can you see a grayed out but ticked Checkbox with the description "Enable Container Tabs" in the Firefox Preferences/Options under "Tabs"? **No**
- Multi-Account Containers Version: **6.1.0**
- Operating System + Version: **KDE Neon 5.14.15**
- Firefox Version: **65.0 (64-bit)**
- Other installed Add-ons + Version + Enabled/Disabled-Status:

Extensions
Name 	Version 	Enabled 	ID
AdBlocker Ultimate	2.37	true	adblockultimate@adblockultimate.net
Checker Plus for Gmail	21.4.9	true	checkerplusforgmail@jasonsavard.com
Facebook Container	1.4.2	true	@contain-facebook
Firefox Multi-Account Containers	6.1.0	true	@testpilot-containers
Googlepanel Lite WE	1.0	true	{796dd796-d98f-44f8-b672-1673cac5db7c}
HTTPS Everywhere	2019.1.31	true	https-everywhere@eff.org
LastPass: Free Password Manager	4.24.0.5	true	support@lastpass.com
Plasma Integration	1.4	true	plasma-browser-integration@kde.org

### Actual behavior
When trying to log into KDE Identities, I get an Incorrect username/password error.

If I do so in regular page, the login works correctly

### Steps to reproduce
1. Open KDE Identities in a container
2. (Assuming one has an account) attempt to log in

**Comment by Kobberholm** [comment]
I cannot reproduce this issue.
I am running KDE Neon 5.15 and Firefox 65 (64 bit) with Plasma Integration 1.4.
When I login, I noticed that it is loading a few fonts from themes.googleusercontent.com, so I figured that might be blocked with the Google Container, but you do not have that and I have tried both with it enabled and disabled.
I would recommend you try creating a blank test profile in Firefox and install just the Multi Account Container and see if that really is where the issue occurs.
